src/youtube/js/youtube.js

```diff
@@ -119,15 +119,19 @@ require("../../../html5-common/js/utils/constants.js");
     playerReady = true;
     for(var i = 0; i < javascriptCommandQueue.length; i++) 
     {
-      if(javascriptCommandQueue[i][0] === "play")
-      { 
-        element.play();
-        hasPlayed = true;
+      switch(javascriptCommandQueue[i][0])
+      {
+        case "play":
+          element.play();
+          hasPlayed = true;
+          break;
+        case "seek":
+          element.seek(javascriptCommandQueue[i][1]);
```

Comment:
Can we add a length/null check to ensure that javascriptCommandQueue[i][1] exists?